packages/protocol/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "@airnode/protocol",
   "version": "0.1.0",
   "license": "MIT",
+  "main": "index",
```

Comment:
I think it's fine to leave this at the package root (?)

Reply:
You mean as opposed to in `src/`? Yeah I think it's fine